packages/synapse-react-client/src/assets/icons/DoubleQuotes.tsx

```diff
@@ -0,0 +1,20 @@
+export const DoubleQuotes = (
```

Comment:
Would prefer if this was just an svg file, since React isn't really doing anything special here.